libpod/image.go

```diff
@@ -121,7 +213,7 @@ func (r *Runtime) PullImage(imgName string, allTags bool, reportWriter io.Writer
 	copyOptions := common.GetCopyOptions(reportWriter, "", nil, nil, common.SigningOptions{})
 
 	for _, image := range images {
-		destRef, err := is.Transport.ParseStoreReference(r.store, image)
+		destRef, err := is.Transport.ParseStoreReference(r.store, srcRef.DockerReference().String())
```

Comment:
Don't you need image here somewhere?

Reply:
nope, this is the destination.  the str i use is correct.